### DISCUSSION TITLE
Scaling Hepatocyte Clearance (Under-prediction & Offset Method)

### DISCUSSION THREAD

**Discussion by paconius** [Open-Systems-Pharmacology/Forum#977] [Questions & Problems]
Hi all,

It is generally acknowledged in the literature that in vitro hepatocyte metabolic stability assays under-predict in vivo hepatic clearance. To compensate for this under-prediction, ADME teams (including those at Merck) will apply an empirical offset correction as part of their best-practices for in-vitro/in-vivo extrapolation. See below for two references that explain this method. 

I am curious if PKSim applies some similar correction method to account for underprediction when using in vitro hepatocyte data.
Or is this concern not relevant to the PKSim approach for some reason?

Yamagata, T., Zanelli, U., Gallemann, D., Perrin, D., Dolgos, H., & Petersson, C. (2017). Comparison of methods for the prediction of human clearance from hepatocyte intrinsic clearance for a set of reference compounds and an external evaluation set. Xenobiotica, 47(9), 741–751. [https://doi.org/10.1080/00498254.2016.1222639](https://doi.org/10.1080/00498254.2016.1222639)

Sohlenius-Sternbeck, A. K., Jones, C., Ferguson, D., Middleton, B. J., Projean, D., Floby, E., Bylund, J., & Afzelius, L. (2012). Practical use of the regression offset approach for the prediction of in vivo intrinsic clearance from hepatocytes. Xenobiotica, 42(9), 841–853. https://doi.org/10.3109/00498254.2012.669080

**Comment by StephanSchaller** [comment]
HI @paconius,

Thanks for your comment.

Indeed, applying an empirical offset correction for in-vitro/in-vivo extrapolation is standard practice, and the same holds for PK-Sim.
But this correction can vary between essays and labs, making it hard to provide a "global" value. 
Thus PK-Sim does not provide such a value.

It is upon the user to "calibrate" a value (or take one from literature) based on the assays and lab procedures of the vendors which were contracted to get such data.

I hope this helps,
Stephan

**Reply by paconius** [reply]
Thank you Stephan. It makes sense that different offsets would be necessary depending on the behavior of a given lab's in vitro assays. 

Within PKSim, what would be the "canonical" way to implement such a correction/offset/calibration? My initial guess is that the user could manually scale the assay t1/2 into CLint,unbound, then apply the offset correction on that value, and finally run all the transformations in reverse until one obtained a "corrected t1/2". This t1/2 value could then be used as input into PKSim. This makes sense, but would be very much opaque and impossible to verify or follow by others inspecting the work. Are you aware of some other cleaner way I could choose to implement in PKSim?

Many thanks for your insights.
